### ISSUE TITLE
FormGroup sub label colour diverges from helper text in dark theme, has low contrast

### ISSUE THREAD

**Issue by berryde** [palantir/blueprint#6807]
#### Environment

- __Package version(s)__: @blueprintjs/core 5.10.3
- __Operating System__: Windows
- __Browser name and version__: Google Chrome 125.0.6422.60

#### Example

Link to a minimal repro: https://blueprintjs.com/docs/#core/components/form-group

#### Steps to reproduce

1. Enable dark theme(⇧D)
1. Enable "Show helper text"
1. Enable "Show sub label"

#### Actual behavior

The helper text and sub label have the same text colour when using the light theme, but a differ in text colour when using the dark theme. The sub label has very low contrast against the background.

![image](https://github.com/palantir/blueprint/assets/56115125/1229575c-37d4-4128-8a3d-1a16750ed28b)

#### Expected behavior

The sub label should have the same colour has the helper text, like when using the light theme:

![image](https://github.com/palantir/blueprint/assets/56115125/6d8ac245-6b89-4eb0-b451-fc00a15a51d3)

#### Possible solution

Update the sub label text colour to match the helper text, like so:

![image](https://github.com/palantir/blueprint/assets/56115125/8d07ee5a-3fb5-4939-b3ae-9961dd6e9eab)
